src/components/ButtonGroup/ButtonGroup.test.js

```diff
@@ -1,17 +1,20 @@
 import React from "react";
 import { mount } from "enzyme";
-import { MemoryRouter } from "react-router-dom";
+import { MemoryRouter, Route } from "react-router-dom";
+import { QueryParamProvider } from "use-query-params";
 
 import ButtonGroup from "./ButtonGroup";
 
 describe("ButtonGroup", () => {
   it("shows active button", () => {
     const wrapper = mount(
       <MemoryRouter>
-        <ButtonGroup
-          buttons={["status", "cloud", "owner"]}
-          activeButton="cloud"
-        />
+        <QueryParamProvider ReactRouterRoute={Route}>
```

Comment:
The `ButtonGroup` component doesn't do anything with the query params or routes so I think this and `MemoryRouter` can be removed in these tests.

Reply:
Removed.